xwiki-platform-core/xwiki-platform-image/xwiki-platform-image-lightbox/xwiki-platform-image-lightbox-test/xwiki-platform-image-lightbox-test-docker/src/test/it/org/xwiki/image/lightbox/test/ui/LightboxIT.java

```diff
@@ -354,6 +354,24 @@ void verifyDownload(TestUtils testUtils, TestReference testReference, TestConfig
         assertEquals(images.get(0), lightboxDownload.getAttribute("download"));
     }
 
+    @Test
+    @Order(12)
+    void verifyPartiallyDisabledLightbox(TestUtils testUtils, TestReference testReference,
+        TestConfiguration testConfiguration)
+    {
+        testUtils.updateObject(LIGHTBOX_CONFIGURATION_REFERENCE, LIGHTBOX_CONFIGURATION_CLASSNAME, 0,
+            "isLightboxEnabled", "1");
```

Comment:
I would move this to a ``enableLightbox(boolean)`` method to prevent code duplication.